src/screens/RequestSwap/SwapScreen.tsx

```diff
@@ -36,15 +38,24 @@ export const SwapScreen = (props: SwapScreenProps) => {
 
```

Comment:
implements uikitten's dropdown select. it is now populated with the appropriate potential swap candidates